BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -155,6 +155,10 @@ public class KeyringStore: ObservableObject {
         if self.selectedAccount.address != selectedAccountAddress {
           if let selectedAccount = selectedAccountKeyring.accountInfos.first(where: { $0.address == selectedAccountAddress }) {
             self.selectedAccount = selectedAccount
+          } else if let firstAccount = selectedAccountKeyring.accountInfos.first {
```

Comment:
this else if is really when `self.selectedAccount.` is just .init() right?

Reply:
Yes it should work for that, and it should also protect against any other _potential_ situation where no selected account exists in the keyring for some coin type. 

FWIW, I cannot hit this flow anymore with the fix from yesterday in #5815, and the fix below to select an account after a keyring is created.

Reply:
can we have some comment on this condition for future reference. i was little confused by looking at it at the beginning

Reply:
There is a comment already below this line